Sources/Build/Sanitizers.swift

```diff
@@ -0,0 +1,108 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2018 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+import Basic
+import Utility
+
+/// A set of enabled runtime sanitizers.
+public struct EnabledSanitizers {
+    public enum Sanitizer: String {
+        case address
+        case thread
+    }
+
+    /// A set of enabled sanitizers.
+    private let sanitizers: [Sanitizer]
```

Comment:
Why'd you change this to array?

Reply:
The simplest data structure that gets the job done.

Reply:
Set was better IMO. The API shouldn't let you express wrong things.

Reply:
The init doesn't allow to express wrong things: https://github.com/apple/swift-package-manager/pull/1536/files#diff-e4fcc1511bb73d0433e0f5d5ac2e442cR30

Though I don't feel strongly about it, e.g. I like how Set<> expresses the intent better.

Reply:
Yeah, I think we should use Set.